mne/time_frequency/tests/test_psd.py

```diff
@@ -82,3 +96,55 @@ def test_psd_epochs():
     assert_true(psds.shape == (1, len(picks), len(freqs)))
     assert_true(np.sum(freqs < 0) == 0)
     assert_true(np.sum(psds < 0) == 0)
+
+def test_compares_psd():
+    """Test PSD estimation on raw for the two computations:
+    plt.psd and the second using scipy.signal.welch
+    """
+    raw = io.Raw(raw_fname)
+
+    exclude = raw.info['bads'] + ['MEG 2443', 'EEG 053']  # bads + 2 more
+
+    # picks MEG gradiometers
+    picks = pick_types(raw.info, meg='grad', eeg=False, stim=False,
+                            exclude=exclude)
+
+    picks = picks[:2]
```

Comment:
you can do it in one line

+    picks = pick_types(raw.info, meg='grad', eeg=False, stim=False,
+                            exclude=exclude)[:2]